src/components/gauge/gauge.scss

```diff
@@ -0,0 +1,222 @@
+@import "../common/_base";
+
+gx-gauge {
+  /**
+  * @prop --component-height:
+   * Set the height or the control. *Must be set before render the control*
+   * (400px by default)
+  */
+  --component-height: 400px;
+
+  /**
+  * @prop --center-circle-text-color:
+   * Define the color of the center text in `circle` gauge type.
+   * (gray by default)
+  */
+  --center-circle-text-color: rgba(0, 0, 0, 0.7);
+
+  /**
+  * @prop --marker-backgorund:
+   * Define the background color of the marker.
+   * (1px by default)
+  */
+  --marker-backgorund: rgba(0, 0, 0, 0.7);
+
+  /**
+  * @prop --max-value-display-background:
+   * Set the color of maximum value display background.
+   * (rgba(44, 44, 44, 1) by default)
+  */
+  --max-value-display-background: rgba(44, 44, 44, 1);
+
+  /**
+  * @prop --max-value-display-color:
+   * Set the color of maximum value display text.
+   * (rgba(255, 255, 255, 0.8) by default)
+  */
+  --max-value-display-color: rgba(255, 255, 255, 0.8);
+
+  /**
+  * @prop --min-value-display-background:
+   * Set the color of minimum value display background.
+   * (rgba(44, 44, 44, 1) by default)
+  */
+  --min-value-display-background: rgba(44, 44, 44, 1);
+
+  /**
+  * @prop --min-value-display-color:
+   * Set the color of minimum value display text.
+   * (rgba(255, 255, 255, 0.8) by default)
+  */
+  --min-value-display-color: rgba(255, 255, 255, 0.8);
+
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  /* justify-content: center;*/
```

Comment:
Please don't forget to remove commented code

Reply:
Comments have been removed in 508735d